app/sentinel/internal/oidc/engine/adapter.go

```diff
@@ -8,16 +8,17 @@
 >/'  SPDX-License-Identifier: BSD-2-Clause
 */
 
-package core
```

Comment:
core is reserved for the only `core` module in the project.

Renamed this as `engine` instead.

cc: @sahinakyol .